pkg/controller/license/list.go

```diff
@@ -5,26 +5,23 @@
 package license
 
 import (
-	"github.com/elastic/cloud-on-k8s/pkg/controller/common/license"
+	"github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1beta1"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
-	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-func listAffectedLicenses(c k8s.Client, licenseName string) ([]reconcile.Request, error) {
-	var list = corev1.SecretList{}
-	// list all cluster licenses referencing the given enterprise license
-	matchLabels := license.NewLicenseByNameSelector(licenseName)
-	err := c.List(&list, matchLabels)
+func reconcileRequestsForAllClusters(c k8s.Client) ([]reconcile.Request, error) {
```

Comment:
nit: this name indicates (to me) that something will be reconciled here

Reply:
It's mean to be read as noun not as verb :-)